build/make/k8s-crd.mk

```diff
@@ -0,0 +1,66 @@
+ARTIFACT_CRD_ID=$(ARTIFACT_ID)-crd
+K8S_HELM_CRD_TARGET ?= $(K8S_RESOURCE_TEMP_FOLDER)/helm-crd
+K8S_HELM_CRD_RESSOURCES ?= k8s/helm-crd
+K8S_HELM_CRD_RELEASE_TGZ=${K8S_HELM_CRD_TARGET}/${ARTIFACT_CRD_ID}-${VERSION}.tgz
+
+K8S_RESOURCE_CRD_COMPONENT ?= "${K8S_RESOURCE_TEMP_FOLDER}/component-${ARTIFACT_CRD_ID}-${VERSION}.yaml"
+K8S_RESOURCE_COMPONENT_CR_TEMPLATE_YAML ?= $(WORKDIR)/build/make/k8s-component.tpl
+
+##@ K8s - CRD targets
+
+.PHONY: crd-helm-generate-chart
+crd-helm-generate-chart: ${K8S_HELM_CRD_TARGET}/Chart.yaml
+
+.PHONY: ${K8S_HELM_CRD_TARGET}/Chart.yaml
+${K8S_HELM_CRD_TARGET}/Chart.yaml: ${BINARY_YQ} $(K8S_RESOURCE_TEMP_FOLDER) k8s-generate
+	@echo "Generate helm crd-chart..."
+	@rm -drf ${K8S_HELM_CRD_TARGET}  # delete folder, so the chart is newly created.
+	@mkdir -p ${K8S_HELM_CRD_TARGET}/templates
+	@cp -r ${K8S_HELM_CRD_RESSOURCES}/** ${K8S_HELM_CRD_TARGET}
+	@${BINARY_YQ} 'select(.kind == "CustomResourceDefinition")' $(K8S_RESOURCE_TEMP_YAML) > ${K8S_HELM_CRD_TARGET}/templates/crd.yaml
```

Comment:
The "normal" yaml is named ArtifactID_Version.yaml, why is this one just called crd.yaml?